packages/@aws-cdk/cfnspec/jest.config.js

```diff
@@ -0,0 +1,10 @@
+const baseConfig = require('../../../tools/cdk-build-tools/config/jest.config');
+module.exports = {
+  ...baseConfig,
+  coverageThreshold: {
+    global: {
+      statements: 70,
+      branches: 50,
+    }
```

Comment:
These thresholds are very low, so I investigated. The original `nodeunit` tests claim to cover 84% statements and 62% branches. However, for some reason `nodeunit` was ignoring code under the `build-tools` folder, of which there is only 1 test (under `build.test.ts`). For the rest of the module, the coverage data shows that the original `nodeunit` tests cover the same lines as the migrated `jest` tests.